scripts/todo.js

```diff
@@ -256,74 +242,76 @@ window.onload = function() {
     const yearCheck = now.getFullYear();
 
     // again, open a transaction then a cursor to iterate through all the data items in the IDB
-    let objectStore = db.transaction(['toDoList'], "readwrite").objectStore('toDoList');
+    const objectStore = db.transaction(['toDoList'], 'readwrite').objectStore('toDoList');
     objectStore.openCursor().onsuccess = function(event) {
```

Comment:
```suggestion
    objectStore.openCursor().onsuccess = (event) => {
```

Reply:
Sure, I will add this to the PR description.